bin/runner.js

```diff
@@ -6,7 +6,11 @@ const url = require('url');
 
 const args = minimist(process.argv.slice(2));
 if (args._.length !== 2 || args.h || args.help) {
-  process.stderr.write('usage: sparql-benchmark-runner endpoint-url path-to-queries [--replication 5] [--warmup 0] [--output output.csv]\n');
+  process.stderr.write('usage: sparql-benchmark-runner endpoint-url path-to-queries [-r 5] [-w 0] [-o output.csv] [--timestamps]\n');
+  process.stderr.write('  -r # of replication runs\n');
```

Comment:
I would add a couple of spaces between param name and description, it reads a bit confusing now.